compiler/rustc_lint/src/internal.rs

```diff
@@ -17,6 +17,10 @@ use rustc_span::symbol::{kw, sym, Symbol};
 use rustc_span::Span;
 
 declare_tool_lint! {
+    /// The `default_hash_type` lint declares use of [`std::collections::HashMap`]/[`std::collections::HashSet`],
```

Comment:
There's something wrong with the "declares"

Reply:
"identifies"?

Reply:
I think I meant to write `detects`, as I did for the others 🤔